index.html

```diff
@@ -1,61 +1,191 @@
-            <strong class="add-product-form__kcal-outcome"></strong>
-            <small class="add-product-form__kcal-info"></small>
-          </div>
+    <link rel="icon" href="icon_code.jpg" />
+  </head>
 
-          <button class="add-product-form__button" type="submit">add</button>
-        </form>
+  <body>
+    <main class="container">
+      <section class="section-box section-box--products">
+        <h2 class="section-box__title">Products</h2>
+        <ul class="product-list"></ul>
       </section>
 
-      <section class="
-              section-box">
-        <h2 class="section-box__title">Today - 06.01.2019</h2>
-      </section>
-    </section>
-  </main>
+      <section class="main-container">
+        <section class="section-box section-box--add-product">
+          <h2 class="section-box__title">Add product</h2>
+
+          <form class="add-product-form">
+            <div class="add-product-form__wrapper">
+              <label class="add-product-form__label" for="product"
+                >Product</label
+              >
+              <select
+                class="add-product-form__dropdown"
+                id="product"
+                name="product"
+              >
+                <option
+                  class="add-product-form__dropdown-item"
+                  disabled
+                  selected
+                  >Choose a product</option
+                >
+              </select>
+            </div>
+
+            <div class="add-product-form__wrapper">
+              <label class="add-product-form__label" for="amount"
+                >Amount(g)</label
+              >
+              <input
+                class="add-product-form__input"
+                id="amount"
+                name="amount"
+                type="number"
+                min="1"
+                step="1"
+                value="100"
+              />
+            </div>
+
+            <div
+              class="add-product-form__wrapper add-product-form__wrapper--result"
+            >
+              <strong class="add-product-form__kcal-outcome"></strong>
+              <small class="add-product-form__kcal-info"></small>
+            </div>
+
+            <button class="add-product-form__button" type="submit">add</button>
+          </form>
+        </section>
+
+        <section
+          class="
+              section-box section-box--summary"
+        >
+          <h2 class="section-box__title">Today - 06.01.2019</h2>
+
+          <div class="section-box__wrapper">
+            <div class="section-box__wrapper--left">
+              <h3 class="section-box__title--left">Chosen products:</h3>
+              <ul class="chosen-product-list">
+                <div class = "chosen-product-list__wrapper">
+                <li
+                  class="chosen-product-list__item"
+                >
+                  <div class="chosen-product-list__item chosen-product-list__item--title">
```

Comment:
Wcześniej tego nie zauważyłem, ale w tych okolicach używasz klasy `chosen-product-list__item` w kontekście `li`, co jest jak najbardziej poprawne, ale też w kontekście tytułu w `div`ie. A tu już coś nie gra, bo to jest semantycznie inny element, mimo modyfikatora `--title`. Zasługuje więc na osobną klasę, może być po prostu `chosen-product-list__item-title`.